x/staking/types/delegation.go

```diff
@@ -126,25 +190,83 @@ func (d UnbondingDelegation) HumanReadableString() (string, error) {
 	resp := "Unbonding Delegation \n"
 	resp += fmt.Sprintf("Delegator: %s\n", d.DelegatorAddr)
 	resp += fmt.Sprintf("Validator: %s\n", d.ValidatorAddr)
-	resp += fmt.Sprintf("Creation height: %v\n", d.CreationHeight)
-	resp += fmt.Sprintf("Min time to unbond (unix): %v\n", d.MinTime)
-	resp += fmt.Sprintf("Expected balance: %s", d.Balance.String())
+	for _, entry := range d.Entries {
```

Comment:
This doesn't return an error. Can we eventually just have `UnbondingDelegation` implement the `Stringer` interface, `String()`?

Reply:
I've changed all these functions in https://github.com/cosmos/cosmos-sdk/pull/3301 to be `String()` for a pretty good reason. Doesn't need to be in this PR.

Reply:
k - leaving for this PR - but agree